data/human/lusk.txt

```diff
@@ -0,0 +1,111 @@
+			choice
+				`	"I take it your meeting didn't go so well?"`
+					goto rant
+				`	"Well, 'hello' to you too."`
+					goto rant
+				`	"Where's my money?"`
+					goto nopay
+			label rant
+			action
+				set "bam: ranted"
+			`	Lusk emits a long, almost exaggerated sigh. "They didn't go for it. This was supposed to be my big break. Those fools don't understand anything. Those idiots. Those... those..." he seems to be working himself up into a lather. "Those... invertebrates don't know the first thing about mining, and yet they see fit to lecture me about prospecting and mineral rights! As if I didn't do my homework? As if I haven't been planning this endeavor for the past six months? I should have known better than to trust these posh clowns with my idea!"`
+			`	You quietly watch what appears to be a man having a breakdown.`
+			`	He suddenly pauses his rant and gives you a wild-eyed look. "You! You would understand, right? I'm sure you've mined plenty of asteroids in your time. You know what it's like to have a surefire cash cow." He doesn't stop to let you correct him. "Why wouldn't they listen? You know what? It doesn't matter. You're a reasonable fellow, aren't you? Will you hear me out?" He finally pauses long enough for you to get a word in edgewise.`
+			choice
+				`	"Listen, Johann. If you just pay me what you owe me, I'll be on my way."`
+					goto nopay
+					to display
+						not "bam: demanded payment"
+				`	"Uh, sure. I'll listen."`
+					goto hearmore
+			label nopay
+			action
+				set "bam: demanded payment"
+			`	For the first time since you met him, Lusk looks somewhat meek. "I... don't have it. The investors at the Ada Entrepreneurial Foundation were supposed to invest in my project. I was going to use some of that money to pay you." He looks down at his feet. "Listen, if you insist on payment right now, I will give you everything that I have." He looks at you with pleading eyes.`
+			choice
+				`	"It's okay. We can discuss that later. What happened at your meeting?"`
+					goto rant
+					to display
+						not "bam: ranted"
+				`	"It's okay. We can discuss that later. I'll hear you out."`
+					goto hearmore
+					to display
+						has "bam: ranted"
+				`	"I'm afraid I must insist."`
+					goto insisted
+				`	"If you don't pay me immediately, I am calling the police."`
+					goto insisted
+			label insisted
+			action
+				payment 9486
+			`	The contrite look on Lusk's face immediately morphs into one of rage. "You want your money, you mewling buffoon? Fine! Here! Take it, and I hope I never see you again!" He rummages hastily through his pockets and pulls out a grimy credit chip before flinging it at your face. You react quickly enough to catch it, but by the time you look back at him, he is already rushing out the door.`
+			`	Upon accessing the credit chip, you find that he only paid you <payment>.`
+				decline
+			label hearmore
```

Comment:
```suggestion
			action
				payment 9486
				clear "bam: ranted"
				clear "bam: demanded payment"
			`	The contrite look on Lusk's face immediately morphs into one of rage. "You want your money, you mewling buffoon? Fine! Here! Take it, and I hope I never see you again!" He rummages hastily through his pockets and pulls out a grimy credit chip before flinging it at your face. You react quickly enough to catch it, but by the time you look back at him, he is already rushing out the door.`
			`	Upon accessing the credit chip, you find that he only paid you <payment>.`
				decline
			label hearmore
			action
				clear "bam: ranted"
				clear "bam: demanded payment"
```
You'll want to clear these conditions unless they're being used in future missions.